loginsightwebhookdemo/__init__.py

```diff
@@ -113,16 +209,21 @@ def introduction():
     ret += Markup("</dl>")
     return ret
 
+
 @app.route("/endpoint/test", methods=['POST'])
-def test():
+@app.route("/endpoint/test/<ALERTID>", methods=['PUT'])
+def test(ALERTID=None):
     """Log the request and respond with success. Don't send the payload anywhere."""
     logging.info(request.get_data())
     return "OK"
 
 
 # Import individual shims
+import loginsightwebhookdemo.hipchat
 import loginsightwebhookdemo.pagerduty
 import loginsightwebhookdemo.pushbullet
+#import loginsightwebhookdemo.servicenow
```

Comment:
Did you intend to include this module? It appears to be a straight copy of the new template.py without being customized for this service. Is that a future pull request?